lib/grape/util/xml.rb

```diff
@@ -0,0 +1,13 @@
+begin
+  require 'multi_xml'
+rescue LoadError
+end
+
+module Grape
+  if Object.const_defined? :MultiXml
+    Xml = ::MultiXml
+  else
+    Xml = ::ActiveSupport::XmlMini
+    Xml::ParseError = StandardError
+  end
```

Comment:
Or are you saying that I should add `require 'active_support/xml_mini'` here after `else`?